frontend/app/root.tsx

```diff
@@ -90,7 +90,7 @@ export default function App() {
 
   useEffect(() => {
     if (adobeAnalytics.isConfigured()) {
-      if (isApplyRoutePathname(location.pathname)) {
+      if (isApplyRouteUrl(document.location.toString())) {
```

Comment:
You're not scalable.

Reply:
> Shouldn't you use `useLocation()` for this instead of relying on the `document` object?

`useLocation` is not a full location URL object, it's pretty minimal.
https://remix.run/docs/en/main/hooks/use-location